### ISSUE TITLE
change request to [vue/require-component-is]

### ISSUE THREAD

**Issue by mesqueeb** [vuejs/eslint-plugin-vue#1295]
**What rule do you want to change?**
- `[vue/require-component-is]`

**Does this change cause the rule to produce more or fewer warnings?**
- fewer

**How will the change be implemented? (New option, new default behavior, etc.)?**
not sure. I think an edit of the source code of this library will be required

**Please provide some example code that this change will affect:**

```vue
<component is="MyInput" >Hello</component>
```

**What does the rule currently do for this code?**
<img width="712" alt="image" src="https://user-images.githubusercontent.com/3253920/92076152-fff0dd80-edf4-11ea-9920-b65f7d318c2a.png">

**What will the rule do after it's changed?**
not throw an error

**Additional context**

So basically, in Vue, props that can accept strings can passed two ways:
- `someprop="string-value"`
- `:someprop="'string-value'"`

These two are identical.

With `<component is` according to the docs, it can receive a component's name as string - of a component that's globally installed with `Vue.component`.

Therefore both these should be allowed by linting behaviour:
- `<component is="MyInput" />`
- `<component :is="'MyInput'" />`

But currently only the latter is allowed.

**Comment by sugoidesune** [comment]
This is what i came here to post. But i didn't come up with this solution 
`<component :is="'MyInput'" />`

I will use this for now.

**Comment by FloEdelmann** [comment]
Why do you want to use `<component is="MyInput" />` instead of `<MyInput />` or `<my-input />`? I think in most cases, omitting the `:` is an actual mistake, so reporting an error seems fine to me.

**Comment by mesqueeb** [comment]
@FloEdelmann sorry I can't remember, I needed to do dynamic things back in those days. Feel free to close.

**Comment by FloEdelmann** [comment]
Alright, thanks for your response. I'll close this then.
